OneCryptoPkg/OneCryptoBin/OneCryptoMmBin.inf

```diff
@@ -0,0 +1,58 @@
+## @file
+#  This module provides OpenSSL Binary implementation.
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = OneCryptoMmBin
```

Comment:
Align naming. I don't have preference between OneCryptoMmBin and OneCryptoBin, but you have both in file structure